documentcloud/documents/models/document.py

```diff
@@ -346,6 +346,21 @@ def page_aspect_ratio(self, page):
 
         return width / height
 
+    def page_size(self, page):
+        "Return the width and height of a given page, as a tuple"
+        default = (8.5, 11.0)
+        if not self.page_spec:
+            return default
+
+        try:
+            dimensions = uncrunch(self.page_spec)[page]
+        except (ValueError, KeyError):
+            return default
+
+        width, height = [float(d) for d in dimensions.split("x")]
+
+        return (width, height)
+
```

Comment:
Adding tests for this new model function, particularly for the error handling paths, would be great.